google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java

```diff
@@ -140,6 +149,13 @@ QueryOptions getDefaultQueryOptions(DatabaseId databaseId) {
     return getOptions().getDefaultQueryOptions(databaseId);
   }
 
+  /**
+   * Returns the {@link ExecutorProvider} to use for async methods that need a background executor.
+   */
+  public ExecutorProvider getAsyncExecutorProvider() {
```

Comment:
how do we set asyncExecutorProvider?

Reply:
It's currently not configurable, and it will automatically use the default provided here: https://github.com/googleapis/java-spanner/blob/b2a56c68695b6209e20f9f86d83d7c5a0f39c7a8/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java#L514

Reply:
thanks for the quick reply Knut Olav .
Is there a plan to make it configurable?

Reply:
There's no concrete plan for it, but this feels like an omission to me. Would you mind submitting a feature request here: https://github.com/googleapis/java-spanner/issues/new?assignees=&labels=&template=feature_request.md

Reply:
thanks so much.
just created [this issue](https://github.com/googleapis/java-spanner/issues/1262)